### DISCUSSION TITLE
[pinned] Introduction to ParlAI discussions

### DISCUSSION THREAD

**Discussion by stephenroller** [facebookresearch/ParlAI#3369] [General]
This forum is a great spot for you to ask general "how do I do X?" type questions in ParlAI. Traditionally, we have handled such questions on Issues, but this forum will be more easily searchable by others with the same questions.

We also welcome general feedback about ParlAI in this forum, or discussion of new ParlAI features that might help your work.

Please obey the [Code of Conduct](https://github.com/facebookresearch/ParlAI/blob/master/CODE_OF_CONDUCT.md) and the GitHub Community Guidelines when discussing in this forum.

Thanks for your interest in ParlAI!

**Comment by evilc3** [comment]
Hey hi I  have used the empathetic bot in interactive mode. But how do I export it to a py file like u would do with tensorflow and pytorch models. 


What I want to do .

1. Get input text from user 
2. Pass it to the parai model 
3. Print the response 

What I know 
1. I need to create a agent 
2. I need to use the observe method 
3. I need to act() 

But I would like to get rid of interactive class can I do that ?

**Reply by stephenroller** [reply]
Go ahead and start a new discussion
